WorkflowTesting/Tests/WorkflowActionTesterTests.swift

```diff
@@ -26,6 +26,16 @@ final class WorkflowActionTesterTests: XCTestCase {
             .verifyState { XCTAssertTrue($0) }
     }
 
+    func test_stateTransitions_throw() throws {
```

Comment:
This tests and those that follow are demonstrative of the testing style these changes support, and serve to ensure we continue to support this testing style. 

If others find unnecessary, I'm happy to remove them.

Reply:
I think this is fine. I don't think it's strictly necessary — it's more of a "ensure the code doesn't change" test than a unit test, but this will help prevent unintended breakages.